sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationKeyValueCollection.cs

```diff
@@ -241,68 +238,6 @@ public virtual Response<AppConfigurationKeyValueResource> Get(string keyValueNam
             }
         }
 
-        /// <summary>
```

Comment:
these methods are removed in this version comparing with our last stable version, which is not allowed.

We need to introduce these methods back in a partial class with the same name and same namespace, please put the partial class in a file with the same name as this file in the `sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Custom` directory.

This partial class should look like this:
```csharp
public partial class AppConfigurationKeyValueCollection : IEnumerable<AppConfigurationKeyValueResource>, IAsyncEnumerable<AppConfigurationKeyValueResource> // previously this class implemented these interfaces, and it must implement them now (and forever)
{
[EditorBrowable(EditorBrowsableState.Never)] // this attribute hides this API from customers, they could not see this
[Obsolete("put a message here", false)] // this attribute marks this API deprecated because it never works
public virtual AsyncPageable<AppConfigurationKeyValueResource> GetAllAsync(string skipToken = null, CancellationToken cancellationToken = default)
{
	throw null;
}
[EditorBrowable(EditorBrowsableState.Never)] // this attribute hides this API from customers, they could not see this
[Obsolete("put a message here", false)] // this attribute marks this API deprecated because it never works
public virtual Pageable<AppConfigurationKeyValueResource> GetAll(string skipToken = null, CancellationToken cancellationToken = default)
{
	throw null;
}

	// we just implement those interfaces by doing nothing
        IEnumerator<AppConfigurationKeyValueResource> IEnumerable<AppConfigurationKeyValueResource>.GetEnumerator()
        {
		yield break;
        }

        IEnumerator IEnumerable.GetEnumerator()
        {
		yield break;
        }

        IAsyncEnumerator<AppConfigurationKeyValueResource> IAsyncEnumerable<AppConfigurationKeyValueResource>.GetAsyncEnumerator(CancellationToken cancellationToken)
        {
		yield break;
        }
}
```

Reply:
Thanks @ArcturusZhang !
I added `Custom\AppConfigurationKeyValueCollection.cs`
Pls let me know if anything else needs to be changed. Thanks!